src/code/code_800D2E30.c

```diff
@@ -0,0 +1,127 @@
+#include <ultra64.h>
+#include <global.h>
+
+extern u8 D_8012DBB0;
+extern PadMgr gPadMgr;
```

Comment:
is this extern needed? we usually reference global variables directly (like gSaveContext) just fine. If it affects codegen then ignore me

Reply:
Doesn't compile without it, I could add it to variables.h

Reply:
yeah if its being used as a global variable then I say is appropriate. Didnt realize it wasnt there already